data/component-design-tokens/checkbox-design-tokens.json

```
@@ -27,41 +27,40 @@
     "value": {
       "width": "{borderWidth.m}",
       "style": "solid",
-      "color": "{color.interactive.error}"
+      "color": "{input.color.border.error}"
     },
     "type": "border"
   },
   "checkbox-border-focus": {
     "value": {
       "width": "{borderWidth.l}",
       "style": "solid",
-      "color": "{color.interactive.focus}"
+      "color": "{input.color.border.focus}"
     },
     "type": "border"
   },
   "checkbox-border-hover": {
     "value": {
       "width": "{borderWidth.m}",
       "style": "solid",
-      "color": "{color.interactive.hover}"
+      "color": "{input.color.border.hover}"
     },
     "type": "border"
   },
```

Missing Border colour on error hover:

```suggestion
  },
  
  "checkbox-border-error-hover": {
    "value": {
      "width": "{borderWidth.m}",
      "style": "solid",
      "color": "{input.color.border.error-hover}"
    },
    "type": "border"
  }
```